omnibus/resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -9,11 +9,11 @@ Try {
 
     if ( test-path $conemulocation )
     {
-        start-process $conemulocation -verb runas -argumentlist '/title',"`"$chefdktitle`"",'/cmd','powershell.exe','-noexit','-command',$chefdkcommand
+        start-process $conemulocation -verb runas -argumentlist '-ExecutionPolicy Bypass','/title',"`"$chefdktitle`"",'/cmd','powershell.exe','-noexit','-command',$chefdkcommand
```

Comment:
These arguments are in the wrong order.  Removing the "-verb runas", we should be left with:
```
        start-process $conemulocation -argumentlist '/title',"`"$chefdktitle`"",'/cmd','powershell.exe','-ExecutionPolicy Bypass','-noexit','-command',$chefdkcommand
```